### ISSUE TITLE
Algolia not indexing relation field, only shows count

### ISSUE THREAD

**Issue by rbsam176** [MattieBelt/mattie-strapi-bundle#139]
I installed the Algolia x Strapi plugin today, everything went really smoothly except for one field. Here's my setup:

On Strapi, I have 2 collection types:
- Guide
- Category

They have a relationship, so a category can belong to many guides.

<img width="831" alt="image" src="https://user-images.githubusercontent.com/51377484/208150866-61d81c21-2ccb-4a8e-aa2e-ef2f45ffc969.png">

When I publish a guide, Algolia seems to only index the count:

<img width="201" alt="image" src="https://user-images.githubusercontent.com/51377484/208151004-1cdbd2bd-56d0-4218-88db-f2cac4d2b2db.png">

My plugins.js search object:
`search: {
        enabled: true,
        config: {
            provider: 'algolia',
            providerOptions: {
                apiKey: env('ALGOLIA_PROVIDER_ADMIN_API_KEY'),
                applicationId: env('ALGOLIA_PROVIDER_APPLICATION_ID'),
            },
              contentTypes: [{ name: 'api::guide.guide' }],
        },
    },`

I've only ever used GraphQL to query Strapi, so I'm not familiar how to specify the content types (eg. api::guide.guide), only that it seems to match the URL in Strapi when viewing a collection type. Is this something to do with the Entity Service API? Does something need to be changed to make it work with relations?

EDIT: I've since learned that in order to make the [REST API](https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/populating-fields.html#field-selection) include relations you need to include a 'populate' parameter, is there something equivalent that can be entered into the contentTypes array in plugins.js?

**Comment by alexanderscott** [comment]
+1, nesting entity relational data into the algolia record payload would be very useful

**Comment by philohelp** [comment]
Yes relations were populated before but it seems that it is no longer the case. It is pretty annoying...

@MattieBelt any idea why this changed ?

**Comment by sauldeleon** [comment]
Hi, spotted the same issue here. Happening with array relationships, like for example, an array of categories belonging to the same post.

**Comment by shadow1349** [comment]
also noticed the same
